session/bootstrap.go

```diff
@@ -407,9 +412,26 @@ func upgrade(s Session) {
 		upgradeToVer23(s)
 	}
 
+	if ver < version24 {
+		upgradeToVer24(s)
+	}
+
 	updateBootstrapVer(s)
 	_, err = s.Execute(context.Background(), "COMMIT")
 
+	timeutil.LoadSystemTZ = func() string {
+		sql := `select variable_value from mysql.tidb where variable_name = "system_tz"`
+		rss, errLoad := s.Execute(context.Background(), sql)
+		if errLoad != nil {
+			log.Fatal(errLoad)
+		}
+		chk := rss[0].NewChunk()
+		rss[0].Next(context.Background(), chk)
+		return chk.GetRow(0).GetString(0)
+	}
+
+	timeutil.LocalStr = timeutil.LoadSystemTZ()
```

Comment:
How about avoiding expose package variable?
Use package function instead.
For example, `timeutil.LocalStr()` is better than `timeutil.LocalStr`, nobody would change the value outside the package.